src/Octokit.Webhooks/Models/CustomPropertyEvent/CustomProperty.cs

```diff
@@ -14,7 +16,25 @@ public sealed record CustomProperty
     public bool Required { get; init; }
 
     [JsonPropertyName("default_value")]
-    public string? DefaultValue { get; init; }
+    public object? Default { get; init; }
+
+    public string? DefaultValue => this.Default switch
+    {
+        string str => str,
+        IEnumerable<string> strings => "[" + string.Join(",", strings) + "]",
+        JsonElement json when json.ValueKind is JsonValueKind.String => json.GetString(),
+        JsonElement json when json.ValueKind is JsonValueKind.Array => "[" + string.Join(",", json.EnumerateArray().Select(e => e.GetString()!)) + "]",
+        _ => null,
+    };
+
+    public IEnumerable<string>? DefaultValues => this.Default switch
+    {
+        string str => [str],
+        IEnumerable<string> strings => strings,
+        JsonElement json when json.ValueKind is JsonValueKind.String => [json.GetString()],
+        JsonElement json when json.ValueKind is JsonValueKind.Array => json.EnumerateArray().Select(e => e.GetString()!),
```

Comment:
Need to supress the null warning here too? `[json.GetString()!]`

Plus a tiny nit:
```suggestion
        JsonElement { ValueKind: JsonValueKind.String } json => [json.GetString()!],
        JsonElement { ValueKind: JsonValueKind.Array } json => json.EnumerateArray().Select(e => e.GetString()!),
```